docs/administrator/security/component-permission.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Component Permissions
+---
+
+# Karmada Component Permissions
+
+As a multi-cluster management framework, it is crucial to control the permissions of Karmada components based on the resources they access to maintain the security and stability of the entire system. The Karmada community has compiled a list of resources that key components require access to and has implemented the principle of least privilege for the relevant components using the community-maintained installation and deployment tool. This document aims to outline the list of resources that Karmada components need to access along with the reasons for accessing those resources, providing clear guidance for users customizing component permissions.
```

Comment:
```suggestion
This document provides a detailed explanation of the resources each Karmada component needs to access and the reasons for these accesses. It will help administrators understand and configure the RBAC permissions needed for Karmada components effectively, ensuring that the system operates securely and efficiently.

The [installation tools](https://karmada.io/docs/installation/) maintained by the community are designed with security in mind. These tools use Role-Based Access Control ([RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)) to manage access to components, ensuring they only have access to the resources they require. By adhering to the principle of least privilege, these tools minimize potential security risks and prevent unauthorized access or actions within the system.
```